consensus/consensus-types/src/timeout_certificate.rs

```diff
@@ -17,6 +17,7 @@ use std::{collections::BTreeMap, fmt};
 pub struct TimeoutCertificate {
     timeout: Timeout,
     signatures: BTreeMap<Author, Ed25519Signature>,
+    markers: BTreeMap<Author, Round>,
```

Comment:
really puzzled: why do we need markers on timeout certificates?

Reply:
Yes, you are right..we don't need markers on TC. Earlier I thought a TC can serve as a QC as well, but it is not the case in the current implementation.